scss/message/_selected-messages-action.scss

```diff
@@ -0,0 +1,16 @@
+.selected-messages-action {
+  display: flex;
+  flex-direction: row;
+  background-color: var(--navBarBackground);
+  justify-content: flex-start;
+  align-items: center;
+  & > p {
+    width: 100%;
+    text-align: right;
+    margin: 8px;
+  }
+  & > button {
+    margin: 4px;
+    margin-bottom: 6px !important;
+  }
```

Comment:
CSS patterns like overriding nested components are tricky and hard to read, the `!important` is indicating that you're overriding some unknown parent style which was defined somewhere else.

If we would use CSS modules and clear custom classes for each version of a button the CSS code would be much easier to read where what is defined & to maintain.

I'm not sure how easy it is to refactor the CSS into modules, but if you can see a way to avoid writing CSS like that with little changes I'd appreciate, otherwise we can look at a proper refactoring later, that's also okay as I don't want to derail your PR 😅

Reply:
yeah a proper refactoring is needed for the buttons component.